conan/internal/model/conf.py

```diff
@@ -101,6 +101,7 @@
     "tools.gnu:pkg_config": "Path to pkg-config executable used by PkgConfig build helper",
     "tools.gnu:build_triplet": "Custom build triplet to pass to Autotools scripts",
     "tools.gnu:host_triplet": "Custom host triplet to pass to Autotools scripts",
+    "tools.gnu:extra_configure_args": "List of extra arguments to pass to configure when using AutotoolsToolchain and GnuToolchain",
```

Comment:
```suggestion
    "tools.gnu:configure_args": "List of extra arguments to pass to configure when using AutotoolsToolchain and GnuToolchain",
```
Seeing there are other ``tools.gnu:xxxx`` configs, probably it is good this way. I'd probably shorten it a bit and remove the ``extra``

Reply:
Done, please, see the commit 91ac324

Reply:
@uilianries @memsharded Why not extra_configure_args? The other ones completely define those public variables, but this only adds extra arguments. IMO, it sounds confusing to me.

Reply:
That's was my first try, to be more aligned to `tools.cmake.cmaketoolchain:extra_variables`

Reply:
Extra variables is because cmaketoolchain already had a way to inject variables before that, the ``user_toolchain``.
But others, like ``tools.build:cxxflags`` are not called "extra", for example, and they are also extending the flags, not overwriting them? 
I somehow assimilated the ``tools.gnu:configure_args`` more to ``tools.build:cxxflags`` than to ``tools.cmake.cmaketoolchain:extra_variables``, but I understand that this is also very subjective, so if you both @franramirez688 and @uilianries feel strong about the ``extra_configure_args``, go for it! :)

Reply:
Renamed again to `tools.cmake.cmaketoolchain:extra_variables`. Thank you both!